test/E2E/conf/githubio.conf.js

```diff
@@ -0,0 +1,35 @@
+const master = require('../../../wdio.conf')
+
+exports.config = Object.assign(master.config, {
+  demo: process.env.DEMO || 'false',
+  githubioTest: 'true',
+  user: process.env.REMOTE_USER || 'user',
```

Comment:
I would prefer if we used `process.env.BROWSERSTACK_USERNAME` and
`process.env.BROWSERSTACK_ACCESS_KEY`, like in the official docs. (the key is not a password)

Reply:
ok